pycqed/instrument_drivers/physical_instruments/_QCC/QCCMicrocode.py

```diff
@@ -0,0 +1,284 @@
+import logging
+import sys
+
+
+def is_number(s):
```

Comment:
`is_number` is a bad number for this function. It actually checks it is a float.